### DISCUSSION TITLE
--turbopack issues with next15 tailwindcss

### DISCUSSION THREAD

**Discussion by sonkiee** [tailwindlabs/tailwindcss#15023] [Help]
⨯ ./src/styles/globals.css:2987:17
Parsing css source code failed
  2985 | }
  2986 |
> 2987 | .peerdata-state=open ~ .peer-\[data-state\=open\]\:hidden {
       |                 ^
  2988 |   display: none;
  2989 | }
  2990 |

Unexpected token Delim('=') at [project]/src/styles/globals.css:2986:16

**Comment by wongjn** [comment]
It looks like you have written a malformed Tailwind class. You may have meant `peer-data-[state=open]:hidden`.

**Reply by sonkiee** [reply]
i didnt write it, i guess tailwind is generating it,  this is what i have on my global.css @tailwind base;
@tailwind components;
@tailwind utilities;

**Reply by wongjn** [reply]
Consider searching in your project, for files covered by the `content` file globs, for the `peer-[data-state=open]:hidden` string. It should be replaced with `peer-data-[state=open]:hidden`.
